doc/api/buffer.md

```diff
@@ -1258,10 +1258,10 @@ changes:
 
 * `value` {string|Buffer|Uint8Array|integer} What to search for.
 * `byteOffset` {integer} Where to begin searching in `buf`. **Default:** `0`.
-* `encoding` {string} If `value` is a string, this is its encoding.
+* `encoding` {string} The encoding of `value` if `value` is a string.
```

Comment:
nit: I’d say something like ``If `value` is a string, this encoding is used to write it into the buffer.``, or something else that doesn’t sound like the encoding is a inherent to the string value itself

Reply:
@addaleax Maybe this? ``` If `value` is a string, this is the encoding used to convert `value` to a binary representation before searching for `value` in `buf` ```?

Reply:
Hmmm...except my wording possibly sounds like `value` is mutated...

Reply:
@Trott Yeah, good point. How about: ``If `value` is a string, this function looks for the representation of `value` in `encoding`.``? It’s less ambiguous and arguably captures the semantics better, but it might also be harder to understand what “representation” means here?